docs/jazzy.yml

```diff
@@ -0,0 +1,92 @@
+module: MapboxNavigation
+author: Mapbox
+author_url: https://www.mapbox.com/navigation-sdk/
+github_url: https://github.com/mapbox/mapbox-navigation-ios
+dash_url: https://www.mapbox.com/navigation-sdk/docsets/Mapbox.xml
+copyright: '© 2014–2017 [Mapbox](https://www.mapbox.com/). See [license](https://github.com/mapbox/mapbox-navigation-ios/blob/master/LICENSE.md) for more details.'
+
+head: |
+  <link rel='shortcut icon' href='https://www.mapbox.com/img/favicon.ico' type='image/x-icon' />
+#skip_undocumented: Yes
+#hide_documentation_coverage: Yes
+
+custom_categories:
+  - name: Guides
+    children:
+      - Storyboards
+  - name: Core Navigation
+    children:
+      - RouteController
+      - RouteControllerDelegate
+      - RouteProgress
+      - RouteLegProgress
+      - RouteStepProgress
+  - name: Drop-in UI
+    children:
+      - NavigationViewController
+      - NavigationViewControllerDelegate
+      - RouteVoiceController
+      - RouteTableViewController
+      - NavigationPulleyViewController
+  - name: Maps
+    children:
+      - NavigationMapView
+      - NavigationMapViewDelegate
+      - MGLMapView
```

Comment:
Yeah, a link inside the documentation comment block would help developers understand that MGLMapView isn’t just a couple methods like the traffic toggle. We could just as well move this entry down into one of the generic sections since NavigationMapView is the important class here.